spec/doublyLinkedList.js

```diff
@@ -0,0 +1,226 @@
+import chai, { expect } from 'chai'
+import chaiChange from 'chai-change'
+import DoublyLinkedList from '../src/doublyLinkedList'
+
+chai.use(chaiChange)
+
+describe('DoublyLinkedList', () => {
+  'use strict'
+
+  it('exists', () => {
+    expect(DoublyLinkedList).to.be.a('function')
+  })
+
+  context('insert()', () => {
+    it('adds a new node to the linked list when empty', () => {
+      const myDoublyLinkedList = new DoublyLinkedList()
```

Comment:
You can put all of these declarations in a beforeEach function after the first describe. Something like: 
let myDoublyLinkedList

beforeEach( () => {
myDoublyLinkedList = new DoublyLinkedList() 
})